src/application-manager.coffee

```diff
@@ -799,7 +799,7 @@ module.exports = class ApplicationManager extends EventEmitter
 				firmware: pathExistsOnHost('/lib/firmware')
 				modules: pathExistsOnHost('/lib/modules')
 			})
-			fs.readFileAsync(path.join(constants.rootMountPoint, '/etc/hostname')).then(String)
+			fs.readFileAsync(path.join(constants.rootMountPoint, '/etc/hostname'), 'utf-8').then(_.trim)
```

Comment:
I don't know if `utf-8` works or not, I've always used `utf8` and it's what the nodejs docs use everywhere: https://nodejs.org/docs/latest-v6.x/api/fs.html

Reply:
Oh, didn't realize that! utf-8 does work, but I changed it to utf8